client/src/shared/App.tsx

```diff
@@ -48,6 +48,7 @@ class App extends React.Component<
   render() {
     let user;
     let isLoggedIn = false;
+    let isEducator = false;
```

Comment:
`isEducator` might be an unused var

Reply:
added it so i wouldnt forget to restrict by userType haha